blockchain/state_manager/src/chain_rand.rs

```diff
@@ -112,7 +112,7 @@ where
             .await
     }
 
-    /// network v0-12; with lookback
+    /// network version 0-12; with look-back
```

Comment:
are you sure about this `look-back`? In the Filecoin spec it's a one word. https://spec.filecoin.io/

Reply:
They're quite fast and loose about creating new words. Lookback isn't a word you would find in a dictionary but rather the combination of two separate words. Ideally I would want the dictionary to only contain new words that have a specific meaning in the filecoin community (like 'Filecoin', 'Merkel', 'tipset', etc).

Reply:
Fine with me.